examples/misc/raptor_master.py

```diff
@@ -224,12 +229,15 @@ def request_cb(self, tasks):
     #
     def result_cb(self, tasks):
 
+        raise RuntimeError('oops')
```

Comment:
@andre-merzky  Why are we raising here? Also, it raises the "unreachable code" error in pylint/flake8